### PR TITLE
Modifying ansible roles for setup-k8s on ubuntu.

### PR DESCRIPTION
**Pull request by dargasudarshan**
Signed-off-by: Sudarshan <sudarshan.darga@cloudbyte.com>

**What this PR does / why we need it**:
- Update ansible roles to use configure-cni.sh instead of configure-weave.sh
- Update ansible vagrantfile to have same hostname and define name, which was impacting the node-failure test.
- Added source profile while executing kubectl commands as part of memleak test.
- Adding token-sha for kubeadm join.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: e2e/ansible/templates/hyperconverged/ubuntu/Vagrantfile.hyperconverged [hunk 1]
```diff
@@ -12,7 +12,7 @@ Vagrant.configure("2") do |config|
 
   # Every Vagrant development environment requires a box. You can search for
   # boxes at https://atlas.hashicorp.com/search.
-  config.vm.box = "ubuntu/xenial64"
+config.vm.box = "openebs/k8s-1.8.8-ubuntu"
```
Comment (yudaykiran) [inline e2e/ansible/templates/hyperconverged/ubuntu/Vagrantfile.hyperconverged]: This change is not required, it is done by setup-env.yml

Comment (dargasudarshan) [reply e2e/ansible/templates/hyperconverged/ubuntu/Vagrantfile.hyperconverged]: Ok @yudaykiran I will revert. This change was done by setup-env.yml

Comment (yudaykiran) [reply e2e/ansible/templates/hyperconverged/ubuntu/Vagrantfile.hyperconverged]: Does it go and modify the file in /ubuntu/Vagrantfile.hyperconverged or the file copied from that location to playbook_dir?

Comment (dargasudarshan) [reply e2e/ansible/templates/hyperconverged/ubuntu/Vagrantfile.hyperconverged]: It will modify the Vagrantfile in playbook_dir location.

Comment (dargasudarshan) [reply e2e/ansible/templates/hyperconverged/ubuntu/Vagrantfile.hyperconverged]: Done. Reverted the changes

**Pull request merged**